homeassistant/components/tesla_fleet/sensor.py

```diff
@@ -378,16 +379,21 @@ class TeslaFleetTimeEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.POWER,
         entity_registry_enabled_default=False,
     ),
-    SensorEntityDescription(
+    TeslaFleetSensorEntityDescription(
         key="island_status",
+        device_class=SensorDeviceClass.ENUM,
         options=[
-            "island_status_unknown",
             "on_grid",
             "off_grid",
             "off_grid_unintentional",
             "off_grid_intentional",
         ],
-        device_class=SensorDeviceClass.ENUM,
+        entity_registry_enabled_default=True,
+        value_fn=lambda value: None if value == "island_status_unknown" else value,
+    ),
+    TeslaFleetSensorEntityDescription(
+        key="storm_mode_active",
+        entity_registry_enabled_default=True,
     ),
```

Comment:
What's the state of this entity?
Also, all entities are enabled by default so that can be omitted

Reply:
I have removed this change from the PR. It should be binary but I'll do that seperately.